Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/GameNetPortal.cs

```diff
@@ -243,6 +244,11 @@ void StartHost()
         {
             SessionManager<SessionPlayerData>.Instance.AddHostData(new SessionPlayerData(NetManager.LocalClientId, PlayerName, m_AvatarRegistry.GetRandomAvatar().Guid.ToNetworkGuid(), 0, true));
             NetManager.StartHost();
+            SceneLoader.Instance.AddOnSceneEventCallback();
+
+            //The "BossRoom" server always advances to CharSelect immediately on start. Different games
+            //may do this differently.
+            SceneLoader.Instance.LoadScene("CharSelect", LoadSceneMode.Single);
```

Comment:
I'm curious why this was moved?

Reply:
It was because it would execute before AddOnSceneEventCallback could be called, but looking back at it I did not need to move it, just to put AddOnSceneEventCallback in the same place instead. I'll move it back, I think it is better how it was.